### ISSUE TITLE
Japan a7riii

### ISSUE THREAD

**Issue by SleepyCiao** [ma1co/Sony-PMCA-RE#441]
I've succeed in changing language from only Japanese to other language like English.
However, once I try to switch the language to some certain languange such as Chinese, my camera reboots.
Is anyone in this trouble?
I'll appreciate it if anyone can help : )

**Comment by dreambold** [comment]
Yes, the issue is expected. You can use this patch, which removes the Chinese option https://github.com/ma1co/Sony-PMCA-RE/pull/356

**Comment by SparkeyLock** [comment]
Maybe a new model or new firmware removes the Chinese font from the bottom of the system.

**Comment by nabstek** [comment]
is unlock language works for firmware version 3.0?
